config/sync/facets.facet.event_categories.yml

```diff
@@ -19,13 +19,13 @@ field_identifier: event_categories
 query_operator: or
 hard_limit: 0
 exclude: false
-use_hierarchy: false
+use_hierarchy: true
 keep_hierarchy_parents_active: false
 hierarchy:
   type: taxonomy
   config: {  }
-expand_hierarchy: false
-enable_parent_when_child_gets_disabled: true
+expand_hierarchy: true
+enable_parent_when_child_gets_disabled: false
```

Comment:
This config name is somewhat a mindfuck...

Reply:
indeed